src/fidesops/service/connectors/saas_connector.py

```diff
@@ -279,9 +291,13 @@ def mask_data(
             query_config.generate_update_stmt(row, policy, privacy_request)
             for row in rows
         ]
+        self.collection_name = node.address.collection
+        update_request: SaaSRequest = self.endpoints[self.collection_name].requests[
+            "update"
+        ]
```

Comment:
I think this should work because if `update` doesn't exist, an error would have already been thrown in `generate_update_stmt` (`get_request_by_action`) -

Reply:
Makes sense to me!